extensions/clusters/kubernetesversions/all.go

```diff
@@ -19,9 +20,9 @@ const (
 	k3sReleasePath      = "v1-k3s-release/releases"
 	gkeVersionPath      = "meta/gkeVersions"
 	aksVersionPath      = "meta/aksVersions"
-	eksVersionsFileURL  = "raw.githubusercontent.com/rancher/ui/master/lib/shared/addon/utils/amazon.js"
+	eksVersionsFileURL  = "raw.githubusercontent.com/rancher/dashboard/refs/heads/master/pkg/eks/assets/data/eks-versions.js"
```

Comment:
Have a question here, for 2.8, are versions still coming from the old one?

Reply:
Yep. The change was introduced from 2.9.